assets/js/modules/analytics/components/setup/SetupForm.js

```diff
@@ -159,6 +159,10 @@ export default function SetupForm( { finishSetup } ) {
 					gtmAnalyticsPropertyID={ gtmAnalyticsPropertyID }
 				/>
 			) }
+			<StoreErrorNotices
+				moduleSlug="analytics-4"
+				storeName={ MODULES_ANALYTICS_4 }
+			/>
```

Comment:
@techanvil Apologies for being very nitpicky here, but do you think it'd be wiser to keep the `<StoreErrorNotices />` adjacent to each other so that just in case both Analytics stores have errors, and there is an existing GTM property notice, the errors wouldn't show up with the notice in middle?

This is a very small change to make, I can go ahead and do it, but I just wanted to run it by you.

Thanks!

Reply:
Hey @nfmohit, thanks for spotting this. No need to apologise at all, it's a very good point! I think this happened as a result of the merge and I just didn't spot it.

I've moved the `StoreErrorNotices` next to each other, with `ExistingGTMPropertyNotice` above them to be consistent with elsewhere.